CMakeLists.txt

```diff
@@ -93,6 +94,10 @@ endif()
 include(ValhallaSanitizerOptions)
 include(ValhallaSourceGroups)
 
+# prefer CONFIG mode over MODULE mode, which versions configuration on the package, not CMake
+# NOTE: this is only supported for cmake >= 3.15, but shouldn't be a problem in real life
+set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
```

Comment:
let's turn this on for everything, it should _always_ be preferred if available as those config modules are shipped with the libraries, instead of cmake